gcp/observability/opencensus.go

```diff
@@ -35,6 +35,19 @@ import (
 var (
 	// It's a variable instead of const to speed up testing
 	defaultMetricsReportingInterval = time.Second * 30
+	o11yMetrics                     = []*view.View{
```

Comment:
I'd rather not put `o11y` into our code base, personally.  It's an abbreviation that many people may not be familiar with.

Should this be `defaultViews` to be even more descriptive?

Reply:
Ah, ok. Kevin said this comment about wordage in my packet as well. Switched.

Reply:
I switched it, but I don't know if I like "default" semantically. These are the views defined by Yash in his document. I don't feel strongly though so went ahead and switched.